src/OSVRTrackedController.h

```diff
@@ -0,0 +1,155 @@
+/** @file
+    @brief OSVR tracked controller
+
+    @date 2015
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+// Copyright 2015 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef INCLUDED_OSVRTrackedController_h_GUID_128E3B29_F5FC_4221_9B38_14E3F402E645
+#define INCLUDED_OSVRTrackedController_h_GUID_128E3B29_F5FC_4221_9B38_14E3F402E645
+
+
+#define NUM_BUTTONS 64
```

Comment:
hum, when I change it to k_EButton_Max & k_unControllerStateAxisCount, after exiting steamvr, Steam crashes!!!  I suspect there might be something up in this code
``` for (int iter_axis = 0; iter_axis < NUM_AXIS; iter_axis++)```
and/or
``` for (int iter_button = 0; iter_button < NUM_BUTTONS; iter_button++) {```
I replaced NUM_AXIS and NUM_BUTTONS with the enum values.